resources/views/layouts/pagination.blade.php

```diff
@@ -23,7 +24,7 @@
                 <li class="page-item">
                     <a class="page-link arrow {{ isset($useAjax) ? 'useAjax' : '' }}"
                         endpoint="{{ route('api.comment.index', ['blogId' => isset($blogId) ? $blogId : '', 'page' => $paginator->currentPage() - 1]) }}"
-                        href="{{ url()->current() }}?{{ http_build_query(['page' => $paginator->currentPage() - 1, 'status' => $status, 'keyword' => $keyword, 'category' => $category]) }}"
+                        href="{{ url()->current() }}?{{ http_build_query(['page' => $paginator->currentPage() - 1, 'status' => $status, 'keyword' => $keyword, 'category' => $category, 'order' => $order]) }}"
```

Comment:
https://stackoverflow.com/questions/24891276/how-to-automatically-append-query-string-to-laravel-pagination-links